modules/write_config.py

```diff
@@ -8,7 +8,7 @@
 init(autoreset=True)
 
 
-def write_config(executor, container, module_results=None):
+def write_config(executor, container, params, module_results=None):
```

Comment:
For consistency with previous PR (#41) should include `cleanup_input`?

```suggestion
def write_config(cleanup_input, executor, container, params, module_results=None):
```

Reply:
Also missing on lines 45-46: 
```
if cleanup_input:
    f.write("cleanup = true\n")
```